tests/wen_new_standard.test.ts

```diff
@@ -0,0 +1,687 @@
+        expect(creator2Data[1]).to.eql("50");
+      });
+      // it.skip("should have wen_royalty_distribution extension registered", async () => {});
+    });
+
+    describe("after freezing", () => {
+      let mintTokenAccountData: Account;
+
+      before(async () => {
+        await program.methods
+          .freezeMintAccount()
+          .accounts({
+            delegateAuthority: mintAuthPublicKey,
+            mint: mintPublicKey,
+            mintTokenAccount,
+            payer: mintAuthPublicKey,
+            user: mintAuthPublicKey,
+          })
+          .preInstructions([
+            createApproveCheckedInstruction(
+              mintTokenAccount,
+              mintPublicKey,
+              mintAuthPublicKey,
+              mintAuthPublicKey,
+              1,
+              0,
+              [],
+              TOKEN_2022_PROGRAM_ID
+            ),
+          ])
+          .rpc({
+            skipPreflight: true,
+            preflightCommitment: "confirmed",
+            commitment: "confirmed",
+          });
+
+        mintTokenAccountData = await getAccount(
+          provider.connection,
+          mintTokenAccount,
+          "confirmed",
+          TOKEN_2022_PROGRAM_ID
+        );
+      });
+
+      it("should be frozen", async () => {
+        expect(mintTokenAccountData.isFrozen).to.be.true;
+      });
+
+      describe("trying to transfer", () => {
+        let logs: string[];
+
+        before(async () => {
+          const transaction = new VersionedTransaction(
+            new TransactionMessage({
+              instructions: [
+                createAssociatedTokenAccountInstruction(
+                  mintAuthPublicKey,
+                  receiverTokenAccount,
+                  receiver.publicKey,
+                  mintPublicKey,
+                  TOKEN_2022_PROGRAM_ID
+                ),
+                createTransferCheckedInstruction(
+                  mintTokenAccount,
+                  mintPublicKey,
+                  receiverTokenAccount,
+                  mintAuthPublicKey,
+                  1,
+                  0,
+                  [],
+                  TOKEN_2022_PROGRAM_ID
+                ),
+              ],
+              payerKey: mintAuthPublicKey,
+              recentBlockhash: (
+                await provider.connection.getLatestBlockhash("confirmed")
+              ).blockhash,
+            }).compileToV0Message()
+          );
+
+          const signedTx = await provider.wallet.signTransaction(transaction);
+          try {
+            await provider.connection.confirmTransaction({
+              ...(await provider.connection.getLatestBlockhash("confirmed")),
+              signature: await provider.connection.sendTransaction(signedTx),
+            });
+          } catch (err) {
+            logs = [
+              err.logs.find(
+                (log: string) => log === "Program log: Error: Account is frozen"
+              ),
+            ];
+          }
+        });
```

Comment:
Just expect that this fails instead checking logs because logs can be truncated.